examples/embed/widget.py

```diff
@@ -103,7 +103,7 @@ def update_pyramid(self):
 pop = Population()
 pop.render()
 
-tag = embed.autoload_server(pop.layout)
+tag = embed.autoload_server(session_id=pop.session.id, pop.layout)
```

Comment:
a bokeh.io.autoload could grab the session id, but for these examples might need an output_server(keep_session_open) or something ...